bucket/gatling.json

```diff
@@ -0,0 +1,36 @@
+{
+    "homepage": "https://gatling.io",
+    "description": "Async Scala-Akka-Netty based Load Test Tool",
+    "version": "3.3.1",
+    "license": "Apache-2.0",
+    "url": "https://repo1.maven.org/maven2/io/gatling/highcharts/gatling-charts-highcharts-bundle/3.3.1/gatling-charts-highcharts-bundle-3.3.1-bundle.zip",
+    "hash": "sha1:6d9630ef1c3af22b217bb7cef3c6179924886469",
+    "extract_dir": "gatling-charts-highcharts-bundle-3.3.1",
+    "bin": [
+        "gatling.bat",
+        "recorder.bat"
+    ],
+    "shortcuts": [
+        [
+            "recorder.bat",
+            "Gatling Recorder"
```

Comment:
Is there icon inside archive, which could be used?

Reply:
As far as I can see, no icons or images are provided in the archive unfortunately